lib/srv/db/common/auth_test.go

```diff
@@ -958,6 +1032,25 @@ func generateAzureVM(t *testing.T, identities []string) armcompute.VirtualMachin
 	}
 }
 
+// generateAzureVM generates Azure scale set VM resource.
+func generateAzureScaleSetVM(t *testing.T, identities []string) armcompute.VirtualMachineScaleSetVM {
```

Comment:
nit: i don't think this function requires `testing.T`. also consider moving to `lib/cloud/mocks`. For example:
https://github.com/gravitational/teleport/blob/c44ac548a029422b0c1ba02c8266062dd10841d5/lib/cloud/mocks/aws_redshift.go#L102-L117